### PR TITLE
srcset fix for gifs. waiting for search

### PR DESCRIPTION
**Pull request by chrisnojima**
some leftover stuff from the larger search pr

@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/icon.desktop.js [hunk 1]
```diff
@@ -59,7 +59,13 @@ export default class Icon extends Component {
 }
 
 const imgName = (type, ext, mult) => `${resolveImageAsURL('icons', type)}${mult > 1 ? `@${mult}x` : ''}.${ext} ${mult}x`
-const imgPath = (type, ext) => [1, 2, 3].map(mult => imgName(type, ext, mult)).join(', ')
+const imgPath = (type, ext) => {
+  if (ext === 'gif') {
```
Comment (chrisnojima) [inline shared/common-adapters/icon.desktop.js]: we don't have @2x type stuff for gifs right now

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits f1695417ebc3...a4dbf53e552a introduced some visual changes on linux:
 * Added: **Search-Waiting**  
   Search-Waiting rendered

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits f1695417ebc3...5be1c07361f6 introduced some visual changes on linux:
 * Added: **Search-Waiting**  
   Search-Waiting rendered

File: shared/search/user-search/render.desktop.js [hunk 2]
```diff
@@ -120,9 +120,9 @@ export function Result ({result, searchText, onClickResult}: {result: SearchResu
 class Render extends Component<void, Props, void> {
   render () {
     return (
-      <Box style={{overflowY: 'auto', flex: 1}}>
+      <Box style={{overflowY: 'auto', flex: 1, position: 'relative', minHeight: 40}}>
         {this.props.waiting && <Icon type='icon-progress-grey-animated'
-          style={{position: 'absolute', top: 20, left: 0, right: 0, marginLeft: 'auto', marginRight: 'auto'}} />}
+          style={{position: 'absolute', width: 20, top: 0, left: 0, right: 0, marginLeft: 'auto', marginRight: 'auto'}} />}
```
Comment (chromakode) [inline shared/search/user-search/render.desktop.js]: Is the `ProgressIndicator` component usable here? The animated icon will cause inconsistency in visdiff; `ProgressIndicator` includes logic to switch to a static version for screenshots.

Comment (chrisnojima) [inline shared/search/user-search/render.desktop.js]: i'll switch it for progressindicator but icon already has provisions for swapping out types so i'm going to move the animated/static logic there so if you use progressindictor or use icon it'll still do the right thing

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits f1695417ebc3...2bd985cd5456 introduced some visual changes on linux:
 * Added: **Search-Waiting**  
   Search-Waiting rendered

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits f0caa342d2e6...378b3f5dc326 introduced some visual changes on linux:
 * Added: **Search-Waiting**  
   Search-Waiting rendered

**Comment by chromakode** [comment]
:+1:

**Pull request merged**